### PR TITLE
Support agent telemetry_list

### PR DESCRIPTION
**Pull request by resdenia**
In agent, we have the option `all_services`(to choose all the params from drop-down in Agent UI), in general integration, we don't have a such resource to monitor.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,98 +1,94 @@
...
+        "substitution_option": "ALLOW_LOOSE"
     }
-  ],
-  "substitutions": {
-    "_PUBSUB_TOPIC_NAME": "logzio_finaltests_is_working_v3-pubsub-topic-logs-to-logzio",
-    "_PUBSUB_SUBSCRIPTION_NAME": "logzio_finaltests_is_working_v3-pubsub-subscription-logs-to-logzio",
-    "_SINK_NAME": "logzio_finaltests_is_working_v3-sink-logs-to-logzio",
-    "_REGION": "us-central1",
-    "_LOGZIO_LISTENER": "listener.logz.io",
-    "_LOGZIO_TOKEN": "vfFDZjscgjhUmUmhafqXaaTKBYNCZwpL",
```
[the remark this answers is not in the document]
Comment (resdenia) [reply config.json:89]: test token was debprecated

File: README.md [hunk 2]
```diff
@@ -76,6 +76,10 @@ Licensed under the [Apache 2.0](http://apache.org/licenses/LICENSE-2.0.txt) Lice
 
 ## Update log
```
Comment (mirii1994) [inline README.md]:
```suggestion
## Changelog
```

File: README.md [hunk 3]
```diff
@@ -76,6 +76,10 @@ Licensed under the [Apache 2.0](http://apache.org/licenses/LICENSE-2.0.txt) Lice
...
 
+**1.2.4**
```
Comment (mirii1994) [inline README.md]:
```suggestion
- **1.2.4**
```
Make the version list a bullet list :)

File: config.json [hunk 4]
```diff
@@ -1,98 +1,94 @@
...
-    {
-      "name": "gcr.io/cloud-builders/gcloud",
-      "args": [
-        "projects",
-        "add-iam-policy-binding",
-        "$PROJECT_ID",
-        "--member=serviceAccount:$PROJECT_NUMBER@cloudbuild.gserviceaccount.com",
-        "--role=roles/logging.admin"
-      ]
+    "steps": [
```
Comment (mirii1994) [inline config.json:2]: In this file only thing that was changed was the token?

**Review by mirii1994 — CHANGES_REQUESTED** [review]

File: config.json (hunk 4 above)
Comment (resdenia) [reply config.json:2]: Rolled back to default values. Token, function name etc.

**Review by mirii1994 — APPROVED** [review]

**Pull request merged**